auth-web/src/components/auth/DecideBusinessInfo.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-container id="decide-business-info-container">
+    <v-row>
+      <!-- Info Column -->
+      <v-col cols="12" md="6">
+        <h2>Decide on a Business Type</h2>
+        <v-list-item class="list-item" v-for="(item, index) in bulletPoints" :key="index" >
+          <v-icon size="6" class="list-item-bullet">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Panel Btns -->
+        <v-btn large outlined color="#003366" class="btn-learn-more"
```

Comment:
Is it possible to use colour variable "$BCgovBlue5" here?

Reply:
For sure, i did try that! But when passing colors into these vue components it didn't actually work when passing it a colour reference from the css theme import. There might be a way, but it might need to be a property that is defined on the component..

Reply:
A style on the v-btn?

Reply:
Updated this in the next commit.
 I noticed i have the 'Learn More Button' in many places, so i just updated the colour to be in the css and made it a component to be re-used in the different components 👍